templates/docs/base/typography.md

```diff
@@ -184,6 +184,14 @@ If you are using the Ubuntu font, it comes in five weights; thin, light, regular
 View example of the Ubuntu font weights.
 </a></div>
 
+### Muted text
+
+To turn text grey, regardless of the tag it is wrapped in, apply the class `p-text--muted`.
```

Comment:
This shouldn't say grey as this could be something different depending on the theme. I would just say "If there is text which is not high priority you can mute it using..." for example.